workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/CodeWriter/Scopes/LoopBlock.cs

```diff
@@ -11,5 +11,15 @@ internal LoopBlock(string declaration, Action<LoopBlock> populate) : base(declar
         {
             populate(this);
         }
+
+        public void Continue()
+        {
+            WriteLine("continue;");
+        }
+
+        public void Break()
+        {
+            WriteLine("break;");
+        }
```

Comment:
These are valid if you are inside a loop, but also if you are inside an if inside a loop. Would I be able to access these inside the if block if there is a loop parent somewhere?

Reply:
(I wouldn't bother reworking things to add this if we can't as I expect it will be complex. Also you can just do since you can just do `Line("continue;`)`. if needed)

Reply:
so a child `if` could access these methods on the parent `loop`, but obviously it would add these statements to the `loop`body and not inside the `if`-body